gateways/token_balances_api_gateway.py

```diff
@@ -51,18 +50,12 @@ def get_token_information(self, token_id: str) -> dict:
                         'field': 'address.keyword'
                     }
                 },
-                'transaction_sum': {
```

Comment:
This is not only not needed as this information will be gathered by using the token index but it is invalid, as the aggregation is being limited by the max aggregation size from elastic search (limited at 1000 rows)